.github/workflows/test_linux_jax_wheels.yml

```diff
@@ -0,0 +1,169 @@
+          - gfx94X-dcgpu
+          - gfx950-dcgpu
+        default: gfx94X-dcgpu
+      release_type:
+        type: choice
+        options: [dev, nightly, prerelease]
+        default: dev
+      rocm_version:
+        type: string
+        required: true
+      tar_url:
+        type: string
+        required: true
+      test_runs_on:
+        description: Runner label to use. The selected runner should have a GPU supported by amdgpu_family
+        required: true
+        type: string
+        default: "linux-mi325-1gpu-ossci-rocm"
+      python_versions:
+        type: string
+        required: true
+        default: "3.12"
+      jax_ref:
+        type: string
+        default: "rocm-jaxlib-v0.7.1"
+      package_index_url:
+        description: Base Python package index URL to test, typically nightly/dev URL with a "v2" or "v2-staging" subdir (without a GPU family subdir)
+        required: true
+        type: string
+        default: "https://rocm.nightlies.amd.com/v2"
+      jax_whl_list:
+        description: list of built jax wheel filenames to fetch & test
+        required: true
+        type: string
+
+permissions:
+  id-token: write
+  contents: read
+  packages: none
+
+jobs:
+  test_jax_wheels:
+    name: Test | ${{ inputs.amdgpu_family }}
+    runs-on: ${{ inputs.test_runs_on }}
+    env:
+      WHEELHOUSE_DIR: ${{ github.workspace }}/jax/wheelhouse
+    steps:
+      - name: Checkout TheRock
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+        with:
+          repository: ${{ inputs.repository || github.repository }}
+          ref: ${{ inputs.ref || '' }}
+
+      - name: Checkout rocm-jax (plugin + build scripts)
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+        with:
+          path: jax
+          repository: rocm/rocm-jax
+          ref: ${{ inputs.jax_ref }}
+
+      - name: Setup Python
+        uses: actions/setup-python@e797f83bcb11b83ae66e0230d6156d7c80228e7c # v6.0.0
+        with:
+          python-version: "3.12"
+
+      - name: Prepare wheelhouse and download JAX wheels
+        shell: bash
+        working-directory: jax
+        run: |
+          python3 ../build_tools/fetch_wheels.py \
+            --cloudfront_url ${{ inputs.package_index_url }} \
+            --amdgpu_family ${{ inputs.amdgpu_family }} \
+            --dir "wheelhouse" \
+            --list_whls '${{ inputs.jax_whl_list }}'
+
+      - name: Compute ROCM_VERSION_SHORT
+        working-directory: jax
+        env:
+            ROCM_VERSION: ${{ inputs.rocm_version }}
+        run: |
+            # Extract major.minor.patch from ROCM_VERSION
+            ROCM_VERSION_SHORT="$(echo "${ROCM_VERSION}" | sed -E 's/^([0-9]+)\.([0-9]+)\.([0-9]+).*/\1.\2.\3/')"
+            echo "ROCM_VERSION_SHORT=${ROCM_VERSION_SHORT}" >> "$GITHUB_ENV"
+            echo "Full ROCm version: ${ROCM_VERSION}"
+            echo "Using semantic ROCm version: ${ROCM_VERSION_SHORT}"
```

Comment:
This code is suspicious and I don't think it should be needed. At a minimum there should be a comment here explaining what problem this code is solving.

Reply:
There is no need to compute a tag. Rather spin up a docker, pip install the jax wheels, run tests and that's it. No need to _build_ a docker container.